docs/device_api.md

```diff
@@ -341,35 +351,92 @@ the following topic:
 ```
 
-## Command payloads
+## Command payloads for UL
+
+Concerning the payload for UL, the command information will have the same information for both transport protocols (HTTP or MQTT).
 
-Concerning the payload, the command information will have the same information for both transport protocols.
 ```
-<device name>@<command name>|<command value>
+<deviceId>@<commandName>|<commandValue>
 ```
-This indicates that the device (named 'device_name' in the Context Broker) has to execute the command 'command_name', with
-the given value. E.g.:
+
+This indicates that the device (which ID is `deviceId`) has to execute the command `commandName`, with the given `commandValue`. E.g.:
+
 ```
 Robot1@turn|left
 ```
+
 This example will tell the Robot 1 to turn to left.
 
-In the case of complex commands requiring parameters, the `command_value` could be used to implement parameter passing. E.g:
+In the case of complex commands requiring parameters, the `commandValue` could be used to implement parameter passing. E.g:
+
 ```
 weatherStation167@ping|param1:1|param2:2
 ```
+
 This example will tell the Weather Station 167 to reply to a ping message with the provided params. Note that `=` cannot be used instead of `:` given that `=` is [a forbidden character for Context Broker](https://fiware-orion.readthedocs.io/en/master/user/forbidden_characters/index.html), so the update at CB triggering the command would be never progressed.
 
-Once the command has finished its execution in the device, the reply to the server must adhere to the following format:
+Once the command has finished its execution in the device, the reply to the IOTA must adhere to the following format:
+
 ```
-<device name>@<command name>|result
+<deviceId>@<commandName>|<result>
 ```
-Where `device_name` and `command_name` must be the same ones used in the command execution, and the result is the
-final result of the command. E.g.:
+
+Where `deviceId` and `commandName` must be the same ones used in the command execution, and `result` is the final result of the command. E.g.:
+
 ```
 weatherStation167@ping|Ping ok
 ```
-In this case, the Weather station replies with a String value indicating everything has worked fine.
+
+In this case, the Weather station replies with a string value indicating everything has worked fine.
+
+## Command payloads for JSON
+
+Concerning the payload for JSON, the command information will have the same information for both transport protocols (HTTP or MQTT).
+
+```
+{
+  "<commandName>": "<commandValue>
+}
+```
+
+This indicates that the device has to execute the command `commandName`, with the given `commandValue`. E.g.:
+
+```
+{
+  "turn": "left"
+}
+```
+
+This example will tell the device to turn to left. Note that different from UL the device id is not included in the payload.
+
+In the case of complex commands requiring parameters, the `commandValue` could be used to implement parameter passing. E.g:
+
+```
+{
+  "ping": "param1:1|param2:2"
```

Comment:
This is not "normative", just ilustrative. I mean, the user can codify parameters has she/he want. Other posibility would be 

```
{
  "ping": "param1:1,param2:2"
}
```

Maybe we should remark that the one shown is just an example, e.g: "We want to remark the above is just an example: user may encode parameters as he/she wants, as long as it complaint with the general syntax defined above" or a similar wording.